.travis.yml

```diff
@@ -47,7 +47,11 @@ env:
       DISTRIB="conda" PYTHON_VERSION="3.5" INSTALL_MKL="true"
       NUMPY_VERSION="1.10.4" SCIPY_VERSION="0.17.0" CYTHON_VERSION="0.23.4"
       CACHED_BUILD_DIR="$HOME/dummy"
-
+    # CI job for PANDAS    
+    - DISTRIB="conda" PYTHON_VERSION="3.5" INSTALL_MKL="true"
```

Comment:
I don't think we want a separate run. Incorporate into the previous run. Please also keep the line length shortish.